src/keys/spend/stealth.rs

```diff
@@ -0,0 +1,90 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use crate::{JubJubAffine, JubJubExtended, PublicKey};
+#[cfg(feature = "canon")]
+use canonical::Canon;
+#[cfg(feature = "canon")]
+use canonical_derive::Canon;
+use dusk_bytes::{DeserializableSlice, Error, HexDebug, Serializable};
+
+use subtle::{Choice, ConstantTimeEq};
+
+/// To obfuscate the identity of the participants, we utilizes a Stealth Address
+/// system.
+/// A `StealthAddress` is composed by a one-time public key (`pk_r`, the actual
+// address) and a random point `R`.
+#[derive(HexDebug, Clone, Copy)]
```

Comment:
Same as above; implement `PartialEq` and `Eq`

Reply:
Is implemented manually due the constant time comparison.